crates/bevy_winit/src/accessibility.rs

```diff
@@ -0,0 +1,206 @@
+use std::{
+    collections::VecDeque,
+    sync::{Arc, Mutex},
+};
+
+use accesskit_winit::Adapter;
+use bevy_a11y::{
+    accesskit::{ActionHandler, ActionRequest, Node, NodeId, Role, TreeUpdate},
+    AccessKitEntityExt, AccessibilityNode, Focus,
+};
+use bevy_app::{App, CoreStage, Plugin};
+use bevy_derive::{Deref, DerefMut};
+use bevy_ecs::{
+    prelude::{DetectChanges, Entity, EventReader, EventWriter},
+    query::With,
+    system::{NonSend, NonSendMut, Query, RemovedComponents, Res, ResMut, Resource},
+};
+use bevy_hierarchy::{Children, Parent};
+use bevy_utils::{default, HashMap};
+use bevy_window::{PrimaryWindow, Window, WindowClosed, WindowFocused};
+
+#[derive(Default, Deref, DerefMut)]
+pub struct AccessKitAdapters(pub HashMap<Entity, Adapter>);
```

Comment:
Doc strings on all of the structs in this module would help readers understand the patterns here.

Module level docs would be a fine substitute.